src/main/java/software/aws/neptune/opencypher/OpenCypherIAMRequestGenerator.java

```diff
@@ -48,28 +51,30 @@ public class OpenCypherIAMRequestGenerator {
      * @param url    URL to point at.
      * @param region Region to use.
      * @return AuthToken for IAM authentication.
-     * @throws SQLException If request cannot be generated.
      */
-    public static AuthToken getSignedHeader(final String url, final String region) throws SQLException {
-        final FullHttpRequest request = new DefaultFullHttpRequest(HttpVersion.HTTP_1_1,
-                io.netty.handler.codec.http.HttpMethod.GET, url);
-        try {
-            new NeptuneNettyHttpSigV4Signer(region, AWS_CREDENTIALS_PROVIDER).signRequest(request);
-        } catch (final NeptuneSigV4SignerException e) {
-            throw SqlError.createSQLException(
-                    LOGGER,
-                    SqlState.CONNECTION_EXCEPTION,
-                    SqlError.FAILED_TO_OBTAIN_AUTH_TOKEN, e);
-        }
+    public static AuthToken createAuthToken(final String url, final String region) {
+        final Request<Void> request = new DefaultRequest<>(SERVICE_NAME);
+        request.setHttpMethod(HttpMethodName.GET);
+        request.setEndpoint(URI.create(url));
+        // Comment out the following line if you're using an engine version older than 1.2.0.0
+        // request.setResourcePath("/opencypher");
+
```

Comment:
Rather than forcing a user to comment out and recompile, should we perhaps allow them to pass in a flag if they are using an older engine version?

Reply:
The comment is for me, not for the user. I'll make another PR as soon as this one is merged, in which I'll uncomment the line above. The README will be updated to show which version of the library is compatible with which Neptune engine version.